app/Models/Location.php

```diff
@@ -28,7 +28,7 @@ class Location extends SnipeModel
         'address2'      => 'max:80|nullable',
         'zip'           => 'min:3|max:10|nullable',
         'manager_id'    => 'exists:users,id|nullable',
-        'parent_id'     => 'nullable|exists:locations,id|different:id',
+        'parent_id'     => 'non_circular:locations,id'
```

Comment:
Can you confirm that this will work with nulled parent_ids? Laravel changed the way their validator worked a while back where you have to specify nullable if it is nullable.

Reply:
(Other than that, this looks great to me :) )